internal/servers/destination/v0/destinations.go

```diff
@@ -186,13 +222,13 @@ func SetDestinationManagedCqColumns(tables []*schema.Table) {
 }
 
 func (s *Server) GetMetrics(context.Context, *pb.GetDestinationMetrics_Request) (*pb.GetDestinationMetrics_Response, error) {
```

Comment:
Do we still need the `GetMetrics` function to be here at all? IIRC we can remove it

Reply:
Oh this is v0, hmm. Ok no it needs to be here, but I think it won't get called since the CLI won't instantiate a v0/v1 server anymore.

Reply:
Yeah. I add `this was deprecated please update CLI`.